gnomad/utils/filtering.py

```diff
@@ -217,7 +217,7 @@ def filter_to_autosomes(
 def add_filters_expr(
     filters: Dict[str, hl.expr.BooleanExpression],
     current_filters: hl.expr.SetExpression = None,
-) -> hl.expr.SetExpression:
+) -> hl.expr:
```

Comment:
Why is this being changed?

Reply:
My PyCharm had warnings about types.

Reply:
Expected type 'SetExpression', got 'Expression' instead: 236. Do you have this too? Or we can just ignore?

Reply:
I get that too, but it's just because the return type of hl.fold is the more general `hl.expr.Expression` since it can also be used on other expression types. However, `current_filters` is `hl.expr.SetExpression` and the output will only ever be `hl.expr.SetExpression`. For example, it is the same as below, which won't have that warning, but they return the same thing:

```
    if current_filters is None:
        current_filters = hl.empty_set(hl.tstr)

    for filter_name, filter_condition in filters.items():
        current_filters = current_filters.union(
            hl.if_else(filter_condition, hl.set([filter_name]), hl.empty_set(hl.tstr))
        )

    return current_filters
```

so I think it's fine as is